kvm/src/lib.rs

```diff
@@ -878,10 +909,35 @@ impl AsRawFd for VcpuFd {
 /// Wrapper for `kvm_cpuid2` which has a zero length array at the end.
 /// Hides the zero length array behind a bounds check.
 #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-#[derive(Clone, Debug, PartialEq)]
 pub struct CpuId {
-    bytes: Vec<u8>,       // Actually accessed as a kvm_cpuid2 struct.
-    allocated_len: usize, // Number of kvm_cpuid_entry2 structs at the end of kvm_cpuid2.
+    /// Wrapper over kvm_cpuid2 from which we only use the first element.
```

Comment:
nit
```suggestion
    /// Wrapper over `kvm_cpuid2` from which we only use the first element.
```
(same for the comment below)